### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by xyz8983**
add read me file
update django version
use env file for configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<b>Note<b>:
+1. please use "_" as delimiters between features/variables name
+
+## Development
+If you would like to contribute to the development, WELCOME!
+
+Please make sure you have `npm` installed.
+We use typescript in this project. Please execute `webpack` to compiles typescript into javascript.
+
+Thorough docstring is required in development
```
Comment (kanastasakis26) [inline README.md]: Maybe, Thorough documentation is required.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+2. viola! you will see a network graph with options to explore with.
+
+<b>Note<b>:
+1. please use "_" as delimiters between features/variables name
+
+## Development
+If you would like to contribute to the development, WELCOME!
+
+Please make sure you have `npm` installed.
+We use typescript in this project. Please execute `webpack` to compiles typescript into javascript.
```
Comment (kanastasakis26) [inline README.md]: compiles -> compile

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+"nodes": [{"index":1, "source": "node_1"}, {"index":2, "source": "node_2"}]
+,
+"links":[{"source": 1, "target": 2, "correlations": 0.58, "link": "node_1_to_2", "support": 300}]
+}
+```
+(the json file can be generated from xxx (will be updated))
+2. viola! you will see a network graph with options to explore with.
+
+<b>Note<b>:
+1. please use "_" as delimiters between features/variables name
```
Comment (kanastasakis26) [inline README.md]: Maybe show an example `feature1_feature2`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    source venv/bin/activate
+    
+    ```
+    
+3. Install required packages
+
+    ```pip install -r requirements.txt```
+4. Create a .env file under the project folder, which contains 
+    ```text
+       SIRIUS_SETTINGS_SECRET_KEY={your string value here with quotes}
```
Comment (kanastasakis26) [inline README.md]: Maybe an explanation of why it is needed and some guidelines for creating the string

Comment (xyz8983) [reply README.md]: sounds good, i will copy pasting from the django doc. so far i m not quite sure how secret key is used.

**Pull request merged**